examples/enable-operator-and-auto-instrumentation/spring-petclinic/spring-petclinic.yaml

```diff
@@ -128,25 +128,13 @@ spec:
           name: admin-server
           ports:
             - containerPort: 9090
+              hostPort: 9090
```

Comment:
hmm, don't think we need to expose host port for these. Let me check why this is added by kompose ( i guess the docker file has it exposed as 9090:9090 but still no reason to expose on k8s node on conversion)

Reply:
Fair, I'll look into it more. Kompose automation is adding this and I would like to keep these demos automatically generated to a certain extend.

Reply:
yeah, maybe just remove it in the script after kompose comvert is run with something like `yq -i 'del(.spec.template.spec.containers[].ports[].hostPort) | del(.spec.template.spec.containers[].ports[].protocol)' file`

Reply:
Done